domains/samanyu.json

```diff
@@ -0,0 +1,14 @@
+{
+  "description": "They call me Soman",
+  "repo": "https://github.com/Samanyu13",
+  "owner": {
+    "username": "Samanyu13",
+    "email": "samanyusaji18@gmail.com"
+  },
+  "record": {
+    "CNAME": "samanyu.dev",
+    "URL": "https://samanyu.is-a.dev"
```

Comment:
`samanyu.dev` doesn't seem to be a valid domain you own.  If you are going to use the domain in the future, I'd suggest using `samanyu.is-a.dev` as a redirection url to `samanyu.dev` instead of using a cname record. Let me know what you want to go with.

Reply:
Oh sorry about that :|. I've updated my CNAME. I guess that'll work.